From 06286cc2b42f71111ab6502eef67cc24a98119a3 Mon Sep 17 00:00:00 2001
From: Scott K Logan <logans@cottsay.net>
Date: Fri, 18 Apr 2025 19:53:59 -0500
Subject: [PATCH] Switch to ament_cmake_ros_core package

---
 CMakeLists.txt | 3 ++-
 package.xml    | 3 ++-
 2 files changed, 4 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 4c4f318..2592f0a 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -10,7 +10,8 @@ if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
   add_compile_options(-Wall -Wextra -Wpedantic)
 endif()
 
-find_package(ament_cmake_ros REQUIRED)
+find_package(ament_cmake REQUIRED)
+find_package(ament_cmake_ros_core REQUIRED)
 find_package(rcpputils REQUIRED)
 find_package(rcutils REQUIRED)
 find_package(gurumdds_cmake_module REQUIRED)
diff --git a/package.xml b/package.xml
index f4815a2..41e0e15 100644
--- a/package.xml
+++ b/package.xml
@@ -9,7 +9,8 @@
   <maintainer email="seontai@gurum.cc">Seontai Kim</maintainer>
   <license>Apache License 2.0</license>
 
-  <buildtool_depend>ament_cmake_ros</buildtool_depend>
+  <buildtool_depend>ament_cmake</buildtool_depend>
+  <buildtool_depend>ament_cmake_ros_core</buildtool_depend>
   <buildtool_depend>rosidl_cmake</buildtool_depend>
 
   <build_depend>gurumdds-3.2</build_depend>
-- 
2.43.0

